src/HealthGPS.Console/command_options.cpp

```diff
@@ -19,75 +19,57 @@ cxxopts::Options create_options() {
     return options;
 }
 
-// NOLINTNEXTLINE(modernize-avoid-c-arrays)
-CommandOptions parse_arguments(cxxopts::Options &options, int &argc, char *argv[]) {
+std::optional<CommandOptions> parse_arguments(cxxopts::Options &options, int argc, char **argv) {
     namespace fs = std::filesystem;
 
     CommandOptions cmd;
-    try {
-        cmd.success = true;
-        cmd.exit_code = EXIT_SUCCESS;
-        cmd.verbose = false;
-        auto result = options.parse(argc, argv);
-        if (result.count("help")) {
-            std::cout << options.help() << '\n';
-            cmd.success = false;
-            return cmd;
-        }
+    cmd.verbose = false;
```

Comment:
Is this line necessary because won't `verbose` get value-initialised to `false` anyway?

Reply:
Nope. The contents of `cmd` are uninitialised at this point, so they could be anything.

If we wrote:

```c++
CommandOptions cmd{};
```

...then it would be. C++ is weird :upside_down_face:

Reply:
Oh right. I thought that when `cmd` got instantiated `verbose` would be value-initialised to `false` because of this:
https://github.com/imperialCHEPI/healthgps/blob/22ad194cfe0e2e62b2f23d92cf9e722d1b379dfa/src/HealthGPS.Console/command_options.h#L24
C++ is weird.

Reply:
Oh, I didn't notice that! I think you're right. I'll remove it.